iron-overlay-behavior.html

```diff
@@ -128,6 +128,14 @@
         value: document
       },
 
+      /**
+       * Set to true to enable restoring of focus when overlay is closed.
+       */
+      restoreFocusOnClose: {
```

Comment:
I thought it would make it clearer when/how this should be used, like for `noCancelOnEscKey` or `noCancelOnOutsideClick`